core/node/proof_data_handler/src/tee_request_processor.rs

```diff
@@ -47,49 +51,52 @@ impl TeeRequestProcessor {
     ) -> Result<Option<Json<TeeProofGenerationDataResponse>>, RequestProcessorError> {
         tracing::info!("Received request for proof generation data: {:?}", request);
 
-        let mut min_batch_number = self.config.tee_config.first_tee_processed_batch;
-        let mut missing_range: Option<(L1BatchNumber, L1BatchNumber)> = None;
+        let batch_ignored_timeout = ChronoDuration::days(10);
```

Comment:
Hardcode or config parameter?

Reply:
I can make it a configurable parameter indeed.

I hardcoded it because I felt it might be confusing in a scenario where, for example, the `batch_ignored_timeout` config parameter is set to `10 days` and later changed to `1 year`. In such a case, it might be unclear at first glance why some entries in the database are marked as `permanently_ignored` even though they are only 10 days old and not at least one year.

Reply:
Fixed [here](https://github.com/matter-labs/zksync-era/pull/2980/commits/0dd01210668c1fb25defa0cbe9723fc5ff3b1abd#diff-9cf7f2a4779fdd713eabf6a97c007aa4114de190846ee8c5e13be555d14c49e3R54-R57). ✅